docs/why-ngrok.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: Why ngrok?
+---
+
+## Environment Independence
+
+ngrok works everywhere. But more than that, it works _the same_ everywhere.
+
+You can run your services in AWS, GCP or Azure. You can run services in your
+on-prem datacenters or on IoT devices like a Raspberry Pi. You can run your
+services on hosted container platforms like Heroku, Render or Fly. You can even
+run services on your laptop. ngrok works the same in any environment and on any
+network.
+
+## Unified Ingress
+
+ngrok consolidates the traditional pile of ingress tools into one platform.
+
+Traditionally, creating ingress for your application services requires
+stitching together multiple different software components. There's a reverse
+proxy, load balancer, API gateway, firewall, delivery network, WAF, caching
+layer, application middleware, DDoS protection and more.
+
+ngrok _collapses_ these components into a platform with unified configuration
+and observability.
+
+## Production-Ready Sooner
+
+ngrok helps you deliver your app to production sooner.
+
+Making your app production-ready requires a lot of setup work to secure it with
+authentication, protect it from attacks and accelerate its traffic so that it's
+fast. ngrok does all of that automatically for you so you can focus on building
+your app and not networking configurations.
+
+## Reusable App Scaffolding
+
+Don't build auth again. Or rate-limiting. Or other undifferentiated
+functionality. You can offload it ngrok instead.
+
+ngrok's module system enables you to build less table-stakes functionality for
+your app and reuse it instead. Spend more time building your software instead
```

Comment:
ngrok's module system enables you to spend less time building table-stakes functionality 
and more time building new features for your app instead.